test/11_array.jl

```diff
@@ -232,6 +234,31 @@ using CategoricalArrays: DefaultRefType, leveltype
     @test x.pool.levels == x2.pool.levels
     @test x.pool.invindex == x2.pool.invindex
 
+    x = y
```

Comment:
Can you add the equivalent code to 12_missingarray.jl? These files should be kept in sync. (Or just leave both files alone as tests in 13_arraycommon.jl may be enough.)

Reply:
added